### PR TITLE
adding elip0002

### PR DESCRIPTION
**Pull request by andreabonel**
Submitting on behalf of @acrual

### CODE REVIEW

**Comment by tiero** [comment]
Isn't this redundant with what Andres already did?

https://github.com/ElementsProject/ELIPs/blob/main/elip-0001.mediawiki

**Comment by apoelstra** [comment]
@tiero no, what I did was copy Bitcoin's BIP2 and basically just search/replaced Bitcoin with Elements. (I did a little bit more than that but not much.)

This ELIP2 was actually written with Elements in mind, and reflects that it has a quite different community and governance structure.

**Comment by tiero** [comment]
Fun there is my name as author 😂  Should @acrual maybe be also the other co-aurthor, I recall he did substantial edits

File: elip-0002.mediawiki [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+
+
+* A protocol ELIP describes any change that affects most or all Elements implementations, such as a change to the network protocol, a change in block or transaction validity rules, or any change or addition that affects the interoperability of applications using Elements.
+* An application ELIP describes a feature addition proposal or design improvement on the application layer of Elements. Proposals do not necessarily represent an Elements community consensus or recommendation, so users and implementers are free to ignore Informational ELIPs or follow their advice.
+
+
+==ELIP Work Flow==
+
+
+The ELIP process begins with a new idea for Elements. Each potential ELIP must have a champion -- someone who writes the ELIP using the style and format described below, shepherds the discussions in the appropriate forums, and attempts to build community consensus around the idea. The ELIP champion (a.k.a. Author) should first attempt to ascertain whether the idea is ELIP-able. Posting to the elements-dev@googlegroups.com mailing list is the best way to go about this.
```
Comment (apoelstra) [inline elip-0002.mediawiki:39]: What is this mailing list? Is it available without dealing with Google?

File: elip-0002.mediawiki [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+It occasionally becomes necessary to transfer ownership of ELIPs to a new champion. In general, we'd like to retain the original author as a co-author of the transferred ELIP, but that's really up to the original author. A good reason to transfer ownership is because the original author no longer has the time or interest in updating it or following through with the ELIP process, or has fallen off the face of the 'net (i.e. is unreachable or not responding to email). A bad reason to transfer ownership is because you don't agree with the direction of the ELIP. We try to build consensus around an ELIP, but if that's not possible, you can always submit a competing ELIP.
+
+
+If you are interested in assuming ownership of an ELIP, send a message asking to take over, addressed to both the original author and the ELIP editor. If the original author doesn't respond to email in a timely manner, the ELIP editor will make a unilateral decision (it's not like such decisions can't be reversed :).
+
+
+==ELIP Editors==
+
+
+The current ELIP editor is Marco Argentieri who can be contacted at marco@vulpem.com
```
Comment (apoelstra) [inline elip-0002.mediawiki:190]: Is Marco willing to take this on? What is his Github name?

Comment (apoelstra) [reply elip-0002.mediawiki:190]: Lol, oh, it's @tiero. Cool.

**Comment by apoelstra** [comment]
This looks good -- it's had a week to sit and nobody has chimed in with any loud complaints. Let's do it.

**Pull request merged**